src/serac/physics/solid.hpp

```diff
@@ -138,12 +170,13 @@ class NonlinearSolid : public BasePhysics {
   void setViscosity(std::unique_ptr<mfem::Coefficient>&& visc_coef);
 
   /**
-   * @brief Set the hyperelastic material parameters
+   * @brief Set the material parameters
    *
-   * @param[in] mu Set the mu Lame parameter for the hyperelastic solid
-   * @param[in] K Set the K Lame parameter for the hyperelastic solid
+   * @param[in] mu Set the shear modulus for the solid
+   * @param[in] K Set the bulk modulus for the solid
+   * @param[in] material_nonlin Flag to include material nonlinearities (linear elastic vs. neo-Hookean model)
    */
-  void setHyperelasticMaterialParameters(double mu, double K);
+  void setMaterialParameters(double mu, double K, bool material_nonlin = true);
```

Comment:
No complaints about this for now, and I assume this will be resolved when material models are fleshed out (as per Jermome's [comment](https://github.com/LLNL/serac/pull/353/files#r570553189)), but I'm curious what the plan is for handling general material models. Something like a `std::vector<double>` of properties that each model can parse, some type of material property container specialized for each model, something else?

Reply:
This will come in @samuelpmishLLNL 's upcoming overhaul. I imagine it will look something like a specialized parameter packet for each material model, but he can speak to it better than I can. Stay tuned!